nix/status-go/build.nix

```diff
@@ -0,0 +1,164 @@
+{ stdenv, lib, fetchFromGitHub, buildGo114Package
+# Dependencies
+, xcodeWrapper
+, go, androidPkgs
+# metadata and status-go source
+, meta, source
+# build parameters
+, platform ? "android"
+, arch ? "386"
+, api ? "23" }:
+
+let
+  inherit (lib) attrNames attrValues getAttr mapAttrs strings concatStringsSep concatMapStrings;
+
+  removeReferences = [ go ];
+  removeExpr = refs: ''remove-references-to ${concatMapStrings (ref: " -t ${ref}") refs}'';
+
+  # Params to be set at build time, important for About section and metrics
+  goBuildParams = {
+    GitCommit = source.rev;
+    Version = source.cleanVersion;
+  };
+
+  # These are necessary for status-go to show correct version
+  paramsLdFlags = attrValues (mapAttrs (name: value:
+    "-X github.com/status-im/status-go/params.${name}=${value}"
+  ) goBuildParams);
+
+  goBuildLdFlags = paramsLdFlags ++ [
+    "-s" # -s disabled symbol table
+    "-w" # -w disables DWARF debugging information
+  ];
```

Comment:
This appears to be unused as well. Using vim's `*` binding is a great way to find those.